### ISSUE TITLE
Automatically show timestamp in user's timezone

### ISSUE THREAD

**Issue by minkcrypto-zz** [uaktags/explorer#38]
We should be able to change GMT timezone to local!

**Comment by uaktags** [comment]
Agreed, this should be a JavaScript feature. I'll start looking into ideas, but data itself should be saved as GMT/native to the coins output. No data-transformation before the save, only a UI transformation

**Comment by uaktags** [comment]
So i'm leaning on the idea that the date/times will be displayed in the user's local time which would be pretty simple to do. Basically run a Date(<dateFromDB>) on it and javascript will do the rest via their browser.

However, I still want the GMT information to be available in some way/shape/form for consistancy purposes. My current idea is to utilize tooltips. On hover, have the GMT date be displayed, so it's there and available. This will remove any issue/fear that there's inconsistancies when you do a GUI view vs a JSONRPC view of the data.
